sklearn/calibration.py

```diff
@@ -238,19 +238,26 @@ class CalibratedClassifierCV(ClassifierMixin, MetaEstimatorMixin, BaseEstimator)
 
     def __init__(
         self,
-        base_estimator=None,
+        estimator=None,
         *,
         method="sigmoid",
         cv=None,
         n_jobs=None,
         ensemble=True,
+        base_estimator="deprecated"
     ):
-        self.base_estimator = base_estimator
+        self.estimator = estimator
         self.method = method
         self.cv = cv
         self.n_jobs = n_jobs
         self.ensemble = ensemble
 
+        if base_estimator != "deprecated":
```

Comment:
Add a test case to check the warning been raised when setting `base_estimator` param.